### PR TITLE
Fix actions not working on touch devices

### LINKED ISSUE #4230: Entity rows not tappable on iOS in 0.102.0b
**Checklist:**
- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Home Assistant release with the issue:**
Home Assistant 0.102.0b1

**Last working Home Assistant release (if known):**
Home Assistant 0.101.3

**UI (States or Lovelace UI?):**
Lovelace

**Browser and Operating System:**
iOS 13.2.2

**Description of problem:**
After installing HA 0.102.0b1 tapping on entity rows to bring up more-info does nothing on iOS. It's like any taps are ignored for those. Some of them I had special `tap_actions` and `hold_actions` assigned to, but most are just plain entities. 

Worked fine in last stable version, and I haven't updated my iOS version since then either, so I do not believe the issue is on Apple's end.

I can reproduce it on the demo website (https://demo.home-assistant.io/) in mobile Safari so it's not an app-specific issue either. Tapping on any of the entity rows there will not bring up more-info. If you tap the _icon_ in the entity row, it does highlight the icon with a circle (which I think is new?). So that shows it's accepting tap input in some cases, just not performing the desired action.

Tapping on other items in the demo like the sensor card ("Temperature study") and thermostat card do bring up more-info as expected, so it seems limited to Entity Row, although I haven't really tested all the other cards to say for sure...

### PR DESCRIPTION
**Pull request by bramkragten**
Fixes https://github.com/home-assistant/home-assistant-polymer/issues/4230

### CODE REVIEW

File: src/panels/lovelace/common/directives/action-handler-directive.ts [hunk 1]
```diff
@@ -193,7 +191,7 @@ class ActionHandler extends HTMLElement implements ActionHandler {
 
 customElements.define("action-handler", ActionHandler);
 
-const geActionHandler = (): ActionHandler => {
+const getActionHandler = (): ActionHandler => {
```
Comment (iantrich) [inline src/panels/lovelace/common/directives/action-handler-directive.ts]: 🤦‍♂

File: src/panels/lovelace/common/directives/action-handler-directive.ts [hunk 2]
```diff
@@ -111,12 +111,10 @@ class ActionHandler extends HTMLElement implements ActionHandler {
...
-      if (options.hasHold) {
-        this.timer = window.setTimeout(() => {
-          this.startAnimation(x, y);
-          this.held = true;
-        }, this.holdTime);
-      }
+      this.timer = window.setTimeout(() => {
+        this.startAnimation(x, y);
+        this.held = true;
+      }, this.holdTime);
```
Comment (iantrich) [inline src/panels/lovelace/common/directives/action-handler-directive.ts]: I remember we discussed this somewhat; something to do with not wanting to show the ripple if there was no hold action set

Comment (bramkragten) [reply src/panels/lovelace/common/directives/action-handler-directive.ts]: Yeah I still think this is not necessary when there is no hold action, but I also know @thomasloven had a lot of trouble getting this to work on all devices, so was not going to experiment 2 days before the release.

Comment (iantrich) [reply src/panels/lovelace/common/directives/action-handler-directive.ts]: absolutely

**Pull request merged**